### PR TITLE
Cart View and Model with add and remove functionality

### PR DESCRIPTION
**Pull request by Laszlo-JFLMTCO**
@njgheorghita @Dpalazzari @drod1000  This pull request closes issue #5  and issue #6 . Already merged locally with latest Master, all tests pass.

### CODE REVIEW

File: app/views/layouts/_flash.html.erb [hunk 1]
```diff
@@ -2,6 +2,6 @@
   <div class="alert alert-<%= name %> alert-dismissable" role="alert">
     <span class="glyphicon glyphicon-exclamation-sign" aria-hidden="true"></span>
     <button type="button" class="close" data-dismiss="alert" aria-label="Close"><span aria-hidden="true">&times;</span></button>
-    <span class="sr-only">Message: </span><%= msg %>
+    <span class="sr-only">Message: </span><%= sanitize(msg, tags: %w(a), attributes: %w(href rel data-method)) %>
```
Comment (njgheorghita) [inline app/views/layouts/_flash.html.erb:5]: nice find, i'm guessing this is basically what html_safe was supposed to do?

**Comment by njgheorghita** [comment]
looks good to me

**Comment by Dpalazzari** [comment]
Ready for mergation

File: app/views/layouts/_flash.html.erb (hunk 1 above)
Comment (Laszlo-JFLMTCO) [reply app/views/layouts/_flash.html.erb:5]: @njgheorghita Sanitize was the key. That is a built-in helper method that allows us to configure what tags and attributes are passed through.

**Pull request merged**